### PR TITLE
[PROFITONOMY-166] Change 404 page appearance

### PR DESCRIPTION
**Pull request by dyudyunov**
Youtrack: https://youtrack.raccoongang.com/issue/SKILLONOMY-166

### CODE REVIEW

File: lms/templates/static_templates/404.html [hunk 1]
```diff
@@ -11,10 +11,10 @@
   <section class="outside-app error404">
     <div class="error404__holder">
       <div class="error404__header-holder">
-        <h1 class="error404__header">${_("Не работают серверы MAMA")}</h1>
+        <h1 class="error404__header">${_("Не работают серверы Profitonomy")}</h1>
```
Comment (OPersian) [inline lms/templates/static_templates/404.html]: How about i18n?

Comment (OPersian) [reply lms/templates/static_templates/404.html]: (I mean updating po/mo)

**Review by bydawen — APPROVED** [review]

**Review by OPersian — APPROVED** [review]

File: lms/templates/static_templates/404.html (hunk 1 above)
Comment (dyudyunov) [reply lms/templates/static_templates/404.html]: Thanks! I'll fix it.

Comment (OPersian) [reply lms/templates/static_templates/404.html]: Doesn't seem important at this point anyway :]

**Pull request merged**